dashboard/config/routes.rb

```diff
@@ -1007,6 +1007,7 @@
     # @see http://guides.rubyonrails.org/routing.html#specifying-constraints
     get '/dashboardapi/v1/districtsearch/:q/:limit', to: 'api/v1/school_districts#search', defaults: {format: 'json'}, constraints: {q: /[^\/]+/}
     get '/dashboardapi/v1/schoolsearch/:q/:limit(/:use_new_search)', to: 'api/v1/schools#search', defaults: {format: 'json'}, constraints: {q: /[^\/]+/}
+    get '/dashboardapi/v1/schoolzipsearch/:q/:limit', to: 'api/v1/schools#zip_search', defaults: {format: 'json'}, constraints: {q: /[^\/]+/}
```

Comment:
Can we make this regex more restrictive??

Reply:
Sorry I should have just reverted this back to a draft once I realized the bigger direction it was going. I definitely will update this regex!